tests/test_calibrateImage.py

```diff
@@ -168,7 +169,7 @@ def _check_run(self, calibrate, result):
 
         # Returned photoCalib should be the applied value, not the ==1 one on the exposure.
         self.assertFloatsAlmostEqual(result.applied_photo_calib.getCalibrationMean(),
-                                     self.photo_calib, rtol=2e-3)
+                                     self.photo_calib, rtol=1e-2)
```

Comment:
Is the higher rtol worth commenting on? Is the actual value close to this or are you just adding some extra safety margin?